lib/optimize/OccurrenceOrderPlugin.js

```diff
@@ -56,12 +60,19 @@ OccurrenceOrderPlugin.prototype.apply = function(compiler) {
 				if(a.identifier() < b.identifier()) return -1;
 				return 0;
 			});
+			// TODO refactor to Map
+			modules.forEach(function(m) {
+				delete m.__OccurenceOrderPlugin_occursInEntry;
+				delete m.__OccurenceOrderPlugin_occurs;
```

Comment:
Avoid using `delete` (detaches the hidden class); probably better to set to `null` or `undefined` instead.

Though this would better approximate a `Map`-based implementation.